### ISSUE TITLE
Granana/Loki : Autocomplete not working for labels starting with upper case letter

### ISSUE THREAD

**Issue by Alan01252** [grafana/grafana#42573]
**What happened**:

When searching in Loki labels container upper case characters renders with a white first letter and auto-complete of the labels values doesn't appear.

![image](https://user-images.githubusercontent.com/112859/144234941-d0bcb721-02df-4bae-a2b7-41467db22469.png)

**What you expected to happen**:

Label name to not have a white first character, and for values for the label to appear.

**How to reproduce it (as minimally and precisely as possible)**:

Create two labels, one starting with UpperCase and one lower case and perform a search.

**Environment**:

- Grafana version: 8.2.5
- Data source type & version: Loki
- OS Grafana is installed on: Ubuntu
- User OS & Browser: Google Chrome

**Comment by aocenas** [comment]
Yeah seems like the syntax assumes only lower case keys for some reason https://github.com/grafana/grafana/blob/15edaa583784c25d1bf53329c6e3f7d9103a2b44/public/app/plugins/datasource/loki/syntax.ts#L194.
